### DISCUSSION TITLE
INSTRUCTIONAL VIDEO - How to Deploy a Plenti site on Netlify from GitLab

### DISCUSSION THREAD

**Discussion by jamestagal** [plentico/plenti#321] [General]
Hi all,

I just released a step-by-step video guide on how to set up and deploy a Plenti site on Netlify. 
Hopefully it is useful to others here. Enjoy!

https://www.youtube.com/watch?v=MQrXo1a0PtU

Ben

**Comment by jimafisk** [comment]
@jamestagal this video is great, thanks so much for putting it together and sharing it with us!!

**Comment by nicosql** [comment]
the docs and default launch content when you fire up Plenti is ... jarringly unorganized & outdated.  It really does the technical achievement of this project a huge disservice in my opinion.  This is a cool concept but I would have to allocate far too much time to training because of the state of the project.

If you have interest in sponsorship I would be interested but I would need a commitment to getting some official updates to the documentation and minimally at least ONE end to end walkthrough (not in a video).

We could do $2k monthly, if that is worth your while please let me know what kind of timeline.

**Reply by jimafisk** [reply]
Hi @nicosql

Thank you for the feedback, I'm definitely aligned with you on improving the documentation and making it easier for folks to get started. 

I would love to get your thoughts on what types of things you'd find helpful covered in the docs and where it would be the most helpful to have this info (whether https://plenti.co/docs, https://github.com/plentico/plenti/wiki, or somewhere else). There are some areas I know we need to flesh out (things like [field widgets](https://plenti.co/docs/field-widgets), custom field widgets, shadow content, etc), but if you have a structure in mind for how to scaffold things to be easier to digest, I'm certainly interested. 

> We could do $2k monthly, if that is worth your while please let me know what kind of timeline.

To be clear, you would be sponsoring the project (not us sponsoring someone to create docs)? That would be huge for us, just let me know what specific examples / updates you'd like to see and we'll get working on it. I've created a discussion to gather ideas: https://github.com/plentico/plenti/discussions/351. Thank you!
